backend/app/Services/WorkOrder/WorkOrderService.php

```
@@ -27,13 +27,19 @@ public function createWorkOrder(array $data): WorkOrder
                 $data['bom_template_ids'] ?? [],
             );
 
+            // Embed an immutable revision block (#180) so the order always shows
+            // which product revision it was released under, even if a newer
+            // revision is released later or the revision is edited/obsoleted.
+            $processSnapshot = $this->attachRevisionSnapshot($processSnapshot, $data['product_revision_id'] ?? null);
+
             // Create work order
             $workOrder = WorkOrder::create([
```

PHPStan (2.2.5)

[error] Call to an undefined static method App\Models\WorkOrder::create().


(staticMethod.notFound)